dumper/README.md

```diff
@@ -7,6 +7,8 @@ logs. This tool generates archives in a format suitable for consumption by the
 
 The Dumper is a Java tool. **[Download the latest cross-platform release zip `dwh-migration-tools-vX.X.X.zip`.](https://github.com/google/dwh-migration-tools/releases/latest)**
 
+The Dumper requires Java 8 or higher. You can check whether you have a recent enough version of Java installed by running the command `java -version`.
```

Comment:
> You can check whether you have a recent enough version of Java installed by running the command `java -version`.

This could be something like 'To check Java version run the command `java -version` or refer to Java vendor documentation"?

Reply:
Done.